src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -244,6 +251,36 @@ internal static void SetupCallbacks(TraceEventDispatcher source)
             clrRundownParser.RuntimeStart += doAtRuntimeStart;
             source.Clr.RuntimeStart += doAtRuntimeStart;
 
+            var symbolParser = new SymbolTraceEventParser(source);
+            symbolParser.ImageIDFileVersion += delegate (FileVersionTraceData data)
+            {
+                TraceProcess process = data.Process();
+
+                if (!string.IsNullOrWhiteSpace(data.OrigFileName) &&
+                    (data.OrigFileName.Equals("clr.dll", StringComparison.OrdinalIgnoreCase) || data.OrigFileName.Equals("mscorwks.dll", StringComparison.OrdinalIgnoreCase))
+                    )
+                {
+                    TraceLoadedDotNetRuntime mang = currentManagedProcess(data); // this will create a mang instance for this process
+                    var version = Version.Parse(data.ProductVersion);
```

Comment:
❓ Is it possible (now or ever) for `data.ProductVersion` to not be a true `Version`? It seems like we should be using `Version.TryParse` here instead to be sure. For example, what happens if a non-.NET process on the machine happens to have a file called **clr.dll** or **mscorwks.dll**?